libqtile/widget/backlight.py

```diff
@@ -82,6 +82,7 @@ class Backlight(base.InLoopPollText):
         ("step", 10, "Percent of backlight every scroll changed"),
         ("format", "{percent:2.0%}", "Display format"),
         ("change_command", "xbacklight -set {0}", "Execute command to change value"),
+        ("min_brightness", 0, "Minimum brightness percentage"),
```

Comment:
I also don't want a totally black screen. I set it to this for two reasons, although I think it can makes sense to have it  default between 1-5. The first is backwards compatibility, as the default was previously 0. The second is the possibility that 0 brightness doesn't mean totally black on all screens.

Open to changing it though, I don't feel strongly either way.

Reply:
Fair enough.